src/pi/flood_detection/package.xml

```diff
@@ -11,6 +11,7 @@
 
   <test_depend>ament_pep257</test_depend>
   <test_depend>ament_mypy</test_depend>
+  <test_depend>python3-pre-commit</test_depend>
```

Comment:
Missing from `pi/camera_streamer` and `pi/pi_info`. Should this be added to those?